### PR TITLE
Add DidJwk and update other DID methods

### PR DESCRIPTION
**Pull request by acekyd**
## What type of PR is this? (check all applicable)

- [x] 📝 Documentation Update
- [x] 🧑‍💻 Code Snippet

## Description

This PR:
- updates the `@web/dids` package
- updates all snippets using `DidxxxMethod` imports to use the newly updated implementations with `Didxxx`
- Add Kotlin and JS snippets on "How to create a DID" including with "DidJwk"
- Updated other snippets especially with issuing VCs with DIDs to the correct code snippets but breaking because of the credentials package explained below

As a result of the `@web5/dids` package update, the following are pending or currently broken and I need more information to fix. 
- `DidIonMethod` uses `generateDwnOptions()` which no longer exists on `DidIon` and I can't find the substitute for it. 
- The credentials package doesn't fully support the updated DIDs and I've spoken with @nitro-neal  and work is underway. 
- `.toKeys()` doesn't currently work in Kotlin to get the Keysets. 

## Added code snippets?
- [x] 👍 yes

## Added tests?

- [x] 👍 yes

## Added to documentation?

- [x] 📓 general documentation

## [optional] What gif best describes this PR or how it makes you feel?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206499975685079
  - https://app.asana.com/0/0/1206499975685082
  - https://app.asana.com/0/0/1206614931786998
  - https://app.asana.com/0/0/1206597543849493

### CODE REVIEW

**Comment by blackgirlbytes** [comment]
Thanks for starting this! This is good to see ahead of time!

File: site/code-snippets/web5/build/decentralized-web-nodes/use-identity-agents.js [hunk 1]
```diff
@@ -18,7 +18,18 @@ export async function getDwnEndpoints() {
 const serviceEndpointNodes = await getTechPreviewDwnEndpoints();
 
 // generates key pairs used for authorization and encryption when interfacing with DWNs
-const didOptions = await DidIonMethod.generateDwnOptions({ serviceEndpointNodes });
+const didOptions = {
+    keySet: {
```
Comment (angiejones) [inline site/code-snippets/web5/build/decentralized-web-nodes/use-identity-agents.js:22]: why do these need to be explicitly defined vs a default set provided by the SDK?

Comment (acekyd) [reply site/code-snippets/web5/build/decentralized-web-nodes/use-identity-agents.js:22]: The `generateDwnOptions` method was removed entirely. In conversation with Moe on what the alternative is, and will update once available.

Comment (angiejones) [reply site/code-snippets/web5/build/decentralized-web-nodes/use-identity-agents.js:22]: @frankhinek the dev shouldn't have to do this right? how do we get around it?

**Review by angiejones — APPROVED** [review]

**Pull request merged**